org.eclipse.swt.gtk.linux.x86/src/org/eclipse/swt/graphics/Color.d

```diff
@@ -51,8 +54,11 @@ public final class Color : Resource {
      * within the packages provided by SWT. It is not available on all
      * platforms and should never be accessed from application code.
      * </p>
+     *
+     * @noreference This field is not intended to be referenced by clients.
      */
     public GdkColor* handle;
+    int alpha = 255;
```

Comment:
This can be `package`, if the Java code doesn't have any explicit protection.

Reply:
Fixed.